Cargo.lock

```diff
@@ -251,15 +260,15 @@ dependencies = [
 
 [[package]]
 name = "bumpalo"
-version = "3.14.0"
+version = "3.15.4"
```

Comment:
This breaks the WASM build. Bumpalo 3.15.0 [changed the MSRV to 1.73.](https://github.com/fitzgen/bumpalo/blob/main/CHANGELOG.md#3150)

Reply:
I would be really happy if more Rust maintainers would consider a compiler version change as breaking... anyway I guess I'll add a CI step for wasm, looks like we're missing them.

Reply:
MSRV bump being considered semver breaking or not seems to be a very contentious issue..
It would be a lot better if cargo update honored the MSRV, I think a lot of issues would be inherently solved by that